Make sure ./configure can eat --with-hc and --with-hc-pkg without giving QA
warnings.

diff -rN -u old-network-2.2.0.0/configure.ac new-network-2.2.0.0/configure.ac
--- old-network-2.2.0.0/configure.ac	2009-07-21 22:07:20.000000000 +0200
+++ new-network-2.2.0.0/configure.ac	2009-07-21 22:07:20.000000000 +0200
@@ -14,6 +14,10 @@
 
 AC_C_CONST
 
+dnl * dummy options to eat --with-hc= and --with-hc-pkg= flags
+AC_ARG_WITH([hc], [(dummy option)], [])
+AC_ARG_WITH([hc-pkg], [(dummy option)], [])
+
 dnl ** check for specific header (.h) files that we are interested in
 AC_CHECK_HEADERS([fcntl.h limits.h stdlib.h sys/types.h unistd.h winsock.h])
 AC_CHECK_HEADERS([arpa/inet.h netdb.h netinet/in.h netinet/tcp.h sys/socket.h sys/uio.h sys/un.h])
